packages/util/src/Constants.ts

```diff
@@ -1 +1,11 @@
-export const ONE_HOUR_MS = 1000 * 60 * 60;
\ No newline at end of file
+export const ONE_HOUR_MS = 1000 * 60 * 60;
+
+/** Email to reach out to for Zupass-related support */
+export const SUPPORT_ZUPASS_EMAIL = "support@zupass.org";
+
+/** Email that will be sending out various notifications or messages related to Zupass */
+export const NOREPLY_ZUPASS_EMAIL = "noreply@zupass.org";
```

Comment:
i think a better name would be `ZUPASS_SENDER_EMAIL` - communicates intent better